packages/material-ui-lab/src/LoadingButton/LoadingButton.js

```diff
@@ -51,22 +50,24 @@ const LoadingButtonRoot = styled(Button, {
       ...(styles.endIconLoadingEnd && {
         [`& .${loadingButtonClasses.endIconLoadingEnd}`]: styles.endIconLoadingEnd,
       }),
-      ...(styles.labelLoadingCenter && {
-        [`& .${loadingButtonClasses.labelLoadingCenter}`]: styles.labelLoadingCenter,
-      }),
     };
   },
-})({
+})(({ styleProps, theme }) => ({
+  transition: theme.transitions.create(['background-color', 'box-shadow', 'border-color'], {
+    duration: theme.transitions.duration.short,
+  }),
   [`& .${loadingButtonClasses.startIconLoadingStart}`]: {
     visibility: 'hidden',
   },
   [`& .${loadingButtonClasses.endIconLoadingEnd}`]: {
     visibility: 'hidden',
   },
-  [`& .${loadingButtonClasses.labelLoadingCenter}`]: {
-    visibility: 'hidden',
-  },
-});
+  ...(styleProps.loadingPosition === 'center' && {
+    [`&.${buttonClasses.disabled}`]: {
+      color: 'transparent',
+    },
+  }),
+}));
```

Comment:
added to fix this.

![image](https://user-images.githubusercontent.com/18292247/121315877-f4f24d00-c932-11eb-95de-fdf99a6ec42a.png)